automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -92,7 +97,7 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   }
 
   def isLoaded: Boolean = {
-    openLaunchAnalysisModalButton.isVisible
+    openLaunchAnalysisModalButton.isVisible && openLaunchAnalysisModalButton.isStateEnabled
```

Comment:
I don't like this. We sometimes come to this page when we expect this button to stay disabled.